### DISCUSSION TITLE
Display of the current IP constantly on the display

### DISCUSSION THREAD

**Discussion by Hotzi57** [beabel/radiowecker#33] [Ideen / Ideas]
If I leave my network with the radio and log in to another network, the radio's IP address is missing to make changes or operations. Searching for the address on the router is only possible on the home device. Hence the idea of showing the current IP address on the display.
As a complete newbie to programming, I managed to do this after a few attempts, not least thanks to “WoKwi”.
To implement this: Insert the following lines into the “tft_display.ino”:
436-tft.setCursor(110,15);
437-tft.setTextColor(ILI9341_WHITE);
438-tft.setTextSize(1);
439-ft.println(WiFi.localIP());

**Comment by beabel** [comment]
In the new Version v2.1.3 IT IS included .
There you See the actaul IP in Touch Mainscreen.

**Comment by Hotzi57** [comment]
Cool, vielen Dank!


kunigunde ***@***.***> schrieb am Mo., 15. Apr. 2024, 19:17:
